Gems/ROS2/Code/Source/Lidar/LidarRaycaster.h

```diff
@@ -15,10 +16,14 @@
 
 namespace ROS2
 {
-    //! A simple implementation of Lidar operation in terms of raycasting.
-    class LidarRaycaster
+    class LidarRaycaster : protected LidarRaycasterRequestBus::Handler
     {
     public:
+        explicit LidarRaycaster(const LidarRaycasterRequestBus::BusIdType& busId);
+        LidarRaycaster(LidarRaycaster&& lidarSystem) noexcept;
+        LidarRaycaster(const LidarRaycaster& lidarSystem) = delete;
+        ~LidarRaycaster() override;
```

Comment:
suggestion: Rule of 5 is good to follow here so we likely want to provide move assignment and copy assignment operators too otherwise this type could break in certain situations

Reply:
Thanks! Fixed.